index.md

```diff
@@ -184,7 +184,9 @@ This is an overview of all Android versions and their corresponding identifiers
   </tr>
   <tr class="table-notes"><td colspan="3">
       <ul>
-        <li><a href="https://developer.android.com/jetpack">Jetpack</a>/<a href="https://developer.android.com/jetpack/androidx">AndroidX</a> libraries <a href="https://developer.android.com/jetpack/androidx/versions#version-table">require</a> a <code>minSdk</code> of 23 or higher since June 2025.</li>              </ul>
+        <li><a href="https://developer.android.com/jetpack">Jetpack</a>/<a href="https://developer.android.com/jetpack/androidx">AndroidX</a> libraries <a href="https://developer.android.com/jetpack/androidx/versions#version-table">require</a> a <code>minSdk</code> of 23 or higher since June 2025.</li>
+        <li>Google Play Services requires API Level 23 or higher <a href="https://9to5google.com/2024/07/11/android-lollipop-google-play-services/">since July 2024</a>.</li>
```

Comment:
Do we have an official blog post/source for this? (See other announcements for play)